packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -99,6 +99,9 @@ export class OdspSummaryUploadManager {
 
             const { url, headers } = getUrlAndHeadersWithAuth(`${this.snapshotUrl}/snapshot`, storageToken);
             headers["Content-Type"] = "application/json";
+            if (parentHandle) {
+                headers["If-Match"] = `fluid:containerid=${parentHandle}`;
```

Comment:
i was just thinking, how can we know if this is being supplied?

Reply:
What we want to do if not supplied? Like we should always make sure that it is supplied looking at the purpose of this?